test.haml

```diff
@@ -0,0 +1,2 @@
+%div.wat{class: "wat"}
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>`%div.wat` can be written as `.wat` since `%div` is implicit<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace